blogging/blog_guidelines.md

```diff
@@ -0,0 +1,131 @@
+# Zowe Blog Guidelines
+
+The blogsite https://medium.com/zowe is administered by the Open Mainframe Project (OMP).  To contact the team responsible for blog contents e-mail [zowe-blog-editors@openmainframeproject.org](email:zowe-blog-editors@openmainframeproject.org).
+
+ - [Blog preparation](#blog-preparation)
+ - [Publishing steps](#publishing-steps)
+   - [When ready to write](#when-ready-to-write)
+   - [When ready to publish](#when-ready-to-publish)
+ - [Content Guidelines](#content-guidelines)
+   - [Zowe Core](#zowe-core)
+   - [Zowe Ecosystem](#zowe-ecosystem)
+   - [Zowe Incubation](#zowe-incubation)
+ - [Personas](#personnas)
+ - [Promotion](#promotion)
+
+## Blog preparation
+
+**Blogs should:**
+
+ - Be brief (generally < 1200; > 500 words)
+ - Focus on a target audience
+ - Be relevant
+   - The title and first paragraph should compel the viewer to take the time to read the piece.  The content should be relevant and helpful for the target audience.
+ - Leverage visuals (screenshots, GIFs, etc.), especially an eye-catching featured image  
+   - See public, license-free sites like unsplash.com
+
+**Discoverability (search results):**
+
+ - Determine which Google search keywords are best suited for your content and target audience (i.e., what are you communicating and to whom?  What keywords are they likely to use?).  Include these keywords in your title and related words in the body, especially the first paragraph.
+
+ - Be sure your title, opening paragraph (and featured image) will stand out in a search by your target audience (i.e., will it get people to click your entry over all the other Google results?)
+
+## Publishing steps
+
+ - Create a Medium handle.  
+ - Let [zowe-blog-editors@openmainframeproject.org](email:zowe-blog-editors@openmainframeproject.org) know your handle so you can be added as a “writer” for the publication.  To see your handle, select your profile picture when you log into medium.com, it will begin with an `@` symbol, e.g. `@myusername_1974.  
+
+### When ready to write
```

Comment:
## Include OMP Branding: 
- At the first mention of Zowe including branding with link's back to the sites: Open Mainframe Project’s Zowe 
- Hyperlink to https://www.openmainframeproject.org/ and https://www.openmainframeproject.org/projects/zowe
- Include the following at the end of your blog: If you enjoyed this blog checkout more Zowe blogs here. Or, join ask a question and join the conversation on the Open Mainframe Project Slack Channel #Zowe-dev, #Zowe-user or #Zowe-onboarding.